worlds/stardew_valley/docs/en_Stardew Valley.md

```diff
@@ -55,20 +69,22 @@ For the locations which do not include a normal reward, Resource Packs and traps
 
 A player can enable some settings that will add some items to the pool that are relevant to progression
 - Seasons Randomizer:
-  - All 4 seasons will be items, and one of them will be selected randomly and be added to the player's start inventory
-  - At the end of each month, the player can choose the next season, instead of following the vanilla season order. On Seasons Randomizer, they can only choose from the seasons they have received.
+  * All 4 seasons will be items, and one of them will be selected randomly and be added to the player's start inventory. 
+  * At the end of each month, the player can choose the next season, instead of following the vanilla season order. On Seasons Randomizer, they can only choose from the seasons they have received.
 - Cropsanity:
-  - Every single seed in the game starts off locked and cannot be purchased from any merchant. Their unlocks are received as multiworld items. Growing each seed and harvesting the resulting crop sends a location check
-  - The way merchants sell seeds is considerably changed. Pierre sells fewer seeds at a high price, while Joja sells unlimited seeds but in huge discount packs, not individually.
+  * Every single seed in the game starts off locked and cannot be purchased from any merchant. Their unlocks are received as multiworld items. Growing each seed and harvesting the resulting crop sends a location check
+  * The way merchants sell seeds is considerably changed. Pierre sells fewer seeds at a high price, while Joja sells unlimited seeds but in huge discount packs, not individually.
```

Comment:
should we clarify that Pierre's stock gets bigger over time or not?

Reply:
Not really, as that feature is purely mod-side, and possibly temporary. I'm still looking for a better design. It'll probably get a LotL episode at some point

Reply:
Ah ok thanks :)